subsys/bluetooth/services/ras/rreq/Kconfig.ras_rreq

```
@@ -17,4 +17,11 @@ module = BT_RAS_RREQ
 module-str = RAS_RREQ
 source "${ZEPHYR_BASE}/subsys/logging/Kconfig.template.log_config"
 
+config BT_RAS_MAX_ACTIVE_RREQ
```

This will match what I've done for RRSP
```suggestion
config BT_RAS_RREQ_MAX_ACTIVE_CONN
```